### ISSUE TITLE
Brave Rewards opt-in ("Start using Brave Rewards") should go through Rewards panel

### ISSUE THREAD

**Issue by Miyayes** [brave/brave-browser#24538]
## Description

Currently, the user can press "Start using Brave Rewards" in:

1. The brave://rewards settings page
2. The tipping banner

And go through a Rewards onboarding flow that takes place in those contexts. 

Instead of doing that, we should funnel all "Start using Brave Rewards" to the Rewards panel by popping it open.

<img src="https://user-images.githubusercontent.com/11497541/183618590-af8d0ba4-3f99-43b9-905c-ea123c4435f4.png" width=400px>

## Solution

- [ ] brave://rewards settings page Rewards opt-in --> Rewards panel
- [ ] Tipping banner Rewards opt-in --> Rewards panel
- [x] Remove all in-page Rewards panel elements from brave://rewards while user is still not opted in, and center the Rewards opt-in component (edit: I think this was resolved for free with another issue, such as https://github.com/brave/brave-browser/issues/16619? cc: @zenparsing)

**Comment by Miyayes** [comment]
Should be completed before/in conjunction with https://github.com/brave/brave-browser/issues/24542.

**Comment by Miyayes** [comment]
For Android, make sure it opens the natively implemented Android Rewards panel.

**Comment by Miyayes** [comment]
>Tipping banner Rewards opt-in --> Rewards panel

@zenparsing It looks like when pressing in-line tipping buttons, it no longer opens the tipping banner, and pops open the Rewards panel directly. That's actually fine to me, but that means it's impossible—i.e., there is no path—to get to the tipping banner *until* one has opted in to Rewards, correct? Again, that's fine to me — just double checking if that is intentional. If so, I can edit the issue description.

**Comment by LaurenWags** [comment]
Verification in progress with

```
Brave | 1.45.87 Chromium: 106.0.5249.91 (Official Build) beta (x86_64)
-- | --
Revision | fa96d5f07b1177d1bf5009f647a5b8c629762157-refs/branch-heads/5249@{#707}
OS | macOS Version 12.6 (Build 21G115)
```
